stl/inc/xthreads.h

```diff
@@ -87,11 +87,11 @@ struct _Cnd_internal_imp_t;
 using _Cnd_t = _Cnd_internal_imp_t*;
 #endif // ^^^ !defined(_M_CEE) ^^^
 
-enum { _Thrd_success, _Thrd_nomem, _Thrd_timedout, _Thrd_busy, _Thrd_error };
+enum class _Thrd_result : int { _Thrd_success, _Thrd_nomem, _Thrd_timedout, _Thrd_busy, _Thrd_error };

```

Comment:
Should we use `_Thrd_result::_Meow` intead of `_Thrd_result::_Thrd_meow`?

I guess the reason why C11 adopted the `thrd_` prefix is that the prefix is weird enough to avoid confliction in almost all cases. We don't need to repeat the _Uglified weird prefix in normal usages.

Reply:
I guess we already have `__std_win_error::_Success` and plenty of identifiers named `_Error`. It's not exactly shadowing so I guess I'd be fine with it.